packages/frontend/src/components/ChartConfigPanel/index.tsx

```diff
@@ -1,15 +1,12 @@
 import { Button } from '@blueprintjs/core';
 import { Popover2 } from '@blueprintjs/popover2';
-import { ChartType } from '@lightdash/common';
 import React from 'react';
 import useEcharts from '../../hooks/echarts/useEcharts';
-import { useVisualizationContext } from '../LightdashVisualization/VisualizationProvider';
 import ChartConfigTabs from './ChartConfigTabs';
 
 export const ChartConfigPanel: React.FC = () => {
-    const { chartType } = useVisualizationContext();
     const eChartsOptions = useEcharts();
-    const disabled = chartType === ChartType.TABLE || !eChartsOptions;
+    const disabled = !eChartsOptions;
```

Comment:
@ZeRego Since ChartConfigPanel is used in SqlRunner as well, don't we need to check for Table chart type as well

Reply:
Nice catch! that's actually related to a different bug. Sql runner isn't showing the table configuration button&panel. 
We have this component in `packages/frontend/src/components/Explorer/VisualizationCard/VisualizationCard.tsx` that makes sure we show the correct button for each type and the sql runner isn't using it. 
```ConfigPanel: FC<{ chartType: ChartType }> = ({ chartType }) => {
    switch (chartType) {
        case ChartType.BIG_NUMBER:
            return <BigNumberConfigPanel />;
        case ChartType.TABLE:
            return <TableConfigPanel />;
        default:
            return <ChartConfigPanel />;
    }
};
```
Pushed a commit to fix it. [abab210](https://github.com/lightdash/lightdash/pull/3146/commits/abab2101684632da0ccdf1d8c036d19ec847791c)

Reply:
Ah yes ! I wondered why that is not using this same func :) makes sense,
Thank you !